report_api/requirements.txt

```diff
@@ -0,0 +1,12 @@
+click==6.7
```

Comment:
awfully strict versions, could these be bit more liberal to allow e.g. minor updates?